pages/common/echo.md

```diff
@@ -14,6 +14,10 @@
 
 `echo -n {{"Hello World"}}`
 
+- Append a message to the file:
+
+`echo {{"Hello World"}} > {{file.txt}}`
```

Comment:
```suggestion
`echo {{"Hello World"}} >> {{file.txt}}`
```
Careful! That command overwrites the file :')

Also, I'd suggest `to a file` instead of `to the file`.

Reply:
I forgot about that. Thanks, @mebeim.

Reply:
Ah, yeah. `to a file` does sound better.